packages/firestore/src/local/local_documents_view.ts

```diff
@@ -288,9 +284,16 @@ export class LocalDocumentsView {
         }
       }
     }
-    return this.remoteDocumentCache.getEntries(
-      transaction,
-      missingBaseDocEntriesForPatching
-    );
+    let mergedDocuments = existingDocuments;
```

Comment:
Nit: please add a newline above this line for readability. It's easier to scan code when it's split into blocks.

Reply:
Done.